src/qibo/models/dbi/utils_scheduling.py

```diff
@@ -0,0 +1,337 @@
+import math
+from copy import deepcopy
+from functools import partial
+from typing import Optional
+
+import hyperopt
+import numpy as np
+
+error = 1e-3
```

Comment:
```suggestion
ERROR= 1e-3
```
In this case since it is a global constant it should be all upper case letters :)